### PR TITLE
[MOB-1414] Replace sections with support portal links

### PR DESCRIPTION
**Pull request by Ayyanchira**
Replaced multiple sections in readme for which the support portal alternative were available. Matched the formatting with iOS readme to maintain consistency.
Added the Iterable Logo as well. :)

### CODE REVIEW

**Comment by Ayyanchira** [comment]
The page looks compact and nice like the one for iOS GitHub read me. 
However, I could not find replacements for the sections:
**Optional setup**
**Migrating from a version prior to 3.1.0** and
**GCM -> Firebase migration**

File: README.md [hunk 1]
```diff
@@ -65,58 +65,6 @@ its usage, read:
...
-    TBD), so it is best to move templates to the `action://` URL scheme as
-    it's possible to do so.
-
-- Consolidated deep link URL handling
-
-    - By default, the SDK handles deep links with the the URL handler
-    assigned to `IterableConfig`. Follow the instructions in
-    [Deep linking](#deep-linking) to migrate any existing URL handling code
-    to this new API.
-
```
Comment (Ayyanchira) [inline README.md:459]: I dont think we can delete this portion as the support docs points to GitHub Readme for help on **Migrating from version prior to 3.1.0**

File: README.md [hunk 2]
```diff
@@ -65,58 +65,6 @@ its usage, read:
...
-### GCM -> Firebase migration
-
-The recommended migration path is to upgrade the existing Google Cloud project to Firebase, update the server token in the existing GCM push integration in Iterable with the new Firebase token, and update the Android app to support Firebase. If you keep using the same project for FCM and the same integration name, the old tokens will still be valid and won't require re-registration of existing devices.
-
-If you're using a different project for FCM and have existing devices on a GCM project with a different sender ID, updating the app will generate new tokens for users, but the old tokens will still be valid. When migrating from one sender ID to another, specify `legacyGCMSenderId` in `IterableConfig` when initializing the SDK. This will disable old tokens to make sure users won't receive duplicate notifications.
```
Comment (Ayyanchira) [inline README.md]: Yes I found a replacement section for this on support portal.. Should we delete this altogether or provide link to GCM -> Firebase migration?

**Comment by roninopf** [comment]
I'll have a corresponding PR open for the migrated docs soon.

**Comment by Ayyanchira** [comment]
Cool then.. I'll merge it.

**Pull request merged**